webrtc.html

```diff
@@ -3147,7 +3230,7 @@ <h2>Methods</h2>
                         current <a>signaling state</a> of <var>connection</var>
                         as described in <span data-jsep=
                         "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>,
-                        then run the following sub-steps:</p>
+                        then run the following sub steps:</p>
```

Comment:
setRemoteDescription(): Don't we need to explicitly guard against description.type being missing now, since it is no longer required?
Otherwise we would end up at "Return the result of setting the RTCSessionDescription to description" with a missing description.

Reply:
Hum yes. I must have been asleep. Will fix. Good catch!